book/website/myst.yml

```diff
@@ -929,6 +929,34 @@ project:
               title: Newsletter's Style Guide
             - file: community-handbook/newsletters/newsletters-template.md
               title: The Turing Way Newsletter Template
+        - file: community-handbook/community-calls.md
```

Comment:
For some reason, the display preview isn't showing the community calls sub-group. I think this might be something with the formatting, let me check!: https://deploy-preview-4122--the-turing-way.netlify.app/community-handbook/community-handbook

Reply:
It does to me. It's using the full title - see screenshot.
<img width="303" alt="image" src="https://github.com/user-attachments/assets/7c43680a-7ca9-453b-94b9-9148701b6d47" />

Reply:
It's working in the preview for me as well :)